### DISCUSSION TITLE
Get min value with pixel coordinate

### DISCUSSION THREAD

**Discussion by nicowha** [ImageMagick/ImageMagick#1933] [General]
Hello everyone!

I am stuck on a script, maybe someone can help me.

I'm currently using ImageMagick 7.0.8-24 Q16 x86_64 command line on OSX

I would like to get the min value of each channel RGB and also get the related pixel coordinate.
I already find a solution for the min (or max) value, but not for the pixel coordinate.
Ultimately each values and pixel coordinate will be store in a unix variable

Maybe someone has a solution?

Thanks

nico

**Comment by fmw42** [comment]
There is a special define that outputs the location of the minimum or maximum pixel(s) of the image. There may be more than one, so there is also a define to limit the number reported.

magick identify -precision 5 -define identify:locate=maximum -define identify:limit=3 image.png

at https://imagemagick.org/script/identify.php

And -define identify:locate=maximum|minimum at https://imagemagick.org/script/command-line-options.php#define

**Reply by nicowha** [reply]
Thanks fmw42!
That's exactly what I was looking for :)
Thank you !

nico
